values/external-dns/external-dns.gotmpl

```diff
@@ -8,9 +8,9 @@ sources:
 provider: {{ $dnsProvider }}
 {{- if eq $dnsProvider "google" }}
 google:
-  project: {{ $v.google.projectId }}
+  project: {{ $v | get "clouds.google.projectId" ""}}
```

Comment:
again, please use `google.projectId` for now

Reply:
done